addons/project/models/project_task.py

```diff
@@ -477,16 +478,24 @@ def _compute_recurring_count(self):
     def _compute_depend_on_count(self):
         tasks_with_dependency = self.filtered('allow_task_dependencies')
         (self - tasks_with_dependency).depend_on_count = 0
+        (self - tasks_with_dependency).closed_depend_on_count = 0
+        if not any(self._ids):
```

Comment:
Shouldn't it be if any record is new, rather than if no record is not new? I guess in practice, it won't make much of a difference, I don't think the compute could be called on a mix of both?